decryptDefaultKey/decryptDefaultKey.ino

```diff
@@ -0,0 +1,82 @@
+#include <AESLib.h>
+
+#define BAUD 9600
+
+AESLib aesLib;
+String kim="r9Huf1ov92qFJMtG";
+// AES Encryption Key
+byte aesKey[] = {kim[0], kim[1], kim[2], kim[3], kim[4], kim[5], kim[6], kim[7], kim[8], kim[9], kim[10], kim[11], kim[12], kim[13], kim[14], kim[15]};
+// General initialization vector (use your own)
+byte aesIv[16] = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+
+// Generate IV (once)
+void aesInit() {
+  aesLib.gen_iv(aesIv);
+  aesLib.set_paddingmode((paddingMode)0);
+  // encrypt("AAAAAAAAAA", aes_iv); // workaround for incorrect B64 functionality on first run... initing b64 is not enough
+
+}
+String decrypt(char * msg, byte iv[]) {
+ 
+  int msgLen = strlen(msg);
+  char decrypted[msgLen]; // half may be enough
+  aesLib.decrypt64(msg, msgLen, decrypted, aesKey, sizeof(aesKey), iv);
+  Serial.println(decrypted);
+  return String(decrypted);
+}
+
+void setup() {
+  Serial.begin(BAUD);
+  while(!Serial);
+  delay(2000);
+  Serial.println("\nBooting...");
+  aesInit();
+}
+
+void logFreeStack(String tag) {
+  #ifdef ESP8266
+  Serial.print("["); Serial.print(tag); Serial.print("] ");
+  Serial.print("free heap: "); Serial.println(ESP.getFreeHeap());
+  #endif
+}
+
+String plaintext = "12345678;";
+int loopcount = 0;
+
+char cleartext[256];
+char ciphertext[512];
+String encrypted="qjGcA6zUUTFkxB7cf/I8/Ot7RsWAPzXxvpjPCySE7GH2nLiIcqTN6Xx304qTExtWMqzVeKrV/yHOYfif1jIsuRhVtF/+9Hp3jMMM1zi3ezS4eOq3om2kwJ7C1ZVZPgIN";
```

Comment:
What are these? expecting only the functions required for encrypt and decrypt. remove all test stuffs. if this is an example add that in an example folder. as per task, we need couple of library functions ( for encrypt/ decrypt) and key generation algorithms

Reply:
ok